android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java

```diff
@@ -469,14 +475,18 @@ private static void putEdges(
     int sizeIndex = media.getColumnIndex(MediaStore.MediaColumns.SIZE);
     int dataIndex = media.getColumnIndex(MediaStore.MediaColumns.DATA);
 
+    boolean shouldGetLocationInfo = includeArrayContains(includeArray, "location");
```

Comment:
As a policy, should we make the includes consistent between platforms? I understand that `filename` and `fileSize` have less performance impact on Android, but I think that leaving it on always will lead to issues like:
> There is no filename on iOS, Android works fine!
Consistently turning both off, gives people full control. By doing a platform check they can modify the includes array to tune it as needed.

Reply:
I agree. I did this, and updated the gifs in the description of the change to show this.